src/audio/src.h

```diff
@@ -31,6 +31,7 @@
 
 #ifndef SRC_H
 #define SRC_H
+#ifdef CONFIG_COMP_SRC
```

Comment:
Not sure what is the reason of pre-processing out the declarations (I guess skipping the compilation of SRC unit should be enough if the relations are ok)? This might make the src code less portable.

Reply:
Ok, I have removed ifdefs from header files